masterlist.yaml

```
@@ -3721,6 +3721,19 @@ plugins:
       - crc: 0x114C9C6A
         util: 'FO4Edit v4.0.2f'
 
+###### Sim Settlements 2 ######
+  - name: 'SS2.esm'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/47976' ]
+    after:
+      - 'HUDFramework.esm'
+      - 'WorkshopFramework.esm'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - '[Extended Dialogue Interface](https://www.nexusmods.com/fallout4/mods/27216)'
+          - '[SS2 - XDI Compatibility Patch](https://www.nexusmods.com/fallout4/mods/48254)'
+        condition: 'active("XDI.esm") and not active("SS2_XDI Patch.esp") or not active("SS2_XDI Patch-Basic.esp") or not active("SS2_XDI Patch-Basic_esl.esp")'
```

These `or`s should be `and`s, otherwise the message will display unless you have all 3 patches active at once.

```suggestion
        condition: 'active("XDI.esm") and not active("SS2_XDI Patch.esp") and not active("SS2_XDI Patch-Basic.esp") and not active("SS2_XDI Patch-Basic_esl.esp")'
```

---

ah, yeah the old negative or. thanks, committed.